doc/deployment/deploy_custom.md

```diff
@@ -11,11 +22,360 @@ This document  details the various options of the `pachctl deploy custom ...` co
 
 ### Preparing your environment
 
-Please see the [introduction to on-premises deployment](./on_premises.html) for steps you need to take prior to creating a Pachydemerm deployment manifest.
+See the [introduction to on-premises deployment](./on_premises.html) for steps that you need to take 
+before creating a custom Pachyderm deployment manifest. 
+That section also explains differences among static persistent volumes, 
+StatefulSets and StatefulSets with StorageClasses, 
+as well as the meanings of the variables, 
+such as  `PVC_STORAGE_SIZE` and `OS_ENDPOINT`, 
+used in examples below.
+
+## Creating a Pachyderm deployment manifest
+
+Use the `pachctl deploy custom` command to create a custom manifest.custom`.
+
+**Example**
+```
+pachctl deploy custom --persistent-disk <persistent disk backend> --object-store <object store backend> \
+    <persistent disk arg1>  <persistent disk arg 2> \
+    <object store arg 1> <object store arg 2>  <object store arg 3>  <object store arg 4> \
+    [[--dynamic-etcd-nodes n] | [--static-etcd-volume <volume name>]]
+    [optional flags]
+```
+
```

Comment:
There should be more of an opening to set the stage here. Something along the lines of:

"The `pachctl deploy custom` command takes a bunch of different flags in order to generate an appropriate manifest for your infrastructure. The flags broadly fall into a few categories:

- `--persistent-disk`: configure the storage resource that is used by etcd. Pachyderm uses etcd to manage administrative metadata. User data is stored in an object store, not etcd.
- `--object-store`: configure the object store that Pachyderm uses for storing all user data that you want versioned and managed.
- `optional flags`: Other configurations and details that can be customized."

These 3 bullets should be the headers for your other sections. They basically already are, so thats good, but this one of those "tell them what you're going to tell them" pieces that some of your big docs are initially missing. In this case you put a TOC to try to offer this, but as svetlana pointed out, it was way too big and confusing when you can just do in a simple intro section as above.